### PR TITLE
Fix js animation not disable if load jquery in body

### PR DESCRIPTION
**Pull request by kuei0221**
Animation disabler now can only turn off the animation come from jQuery if jQuery is loaded in head, because `insert_disable` place the template at the end of head.
https://github.com/teamcapybara/capybara/blob/a08e0477ac2cfd3319d8a423d8661b73103acbf3/lib/capybara/server/animation_disabler.rb#L42-L44

If jQuery is loaded with the body, jQuery will be undefined in head so the function won't work.
https://github.com/teamcapybara/capybara/blob/a08e0477ac2cfd3319d8a423d8661b73103acbf3/lib/capybara/server/animation_disabler.rb#L46-L49

By placing it to the end of the body, developer do not have to worry about where they are loading jQuery and give a better experience 😄

### CODE REVIEW

File: spec/shared_selenium_session.rb [hunk 1]
```diff
@@ -405,7 +405,9 @@
           @animation_session.visit('with_animation')
           scroll_y = @animation_session.evaluate_script(<<~JS)
             (function(){
-              window.scrollTo(0,500);
+              $('html, body').animate({
```
Comment (twalpole) [inline spec/shared_selenium_session.rb:408]: @Why are you modifying this test?

File: lib/capybara/spec/views/with_animation.erb [hunk 2]
```diff
@@ -77,6 +63,20 @@
     <a href='#' class='animation' onclick='this.classList.add("away")' oncontextmenu='this.classList.add("pseudo")'>
       animate me away
     </a>
+    <script src="/jquery.js" type="text/javascript" charset="utf-8"></script>
```
Comment (twalpole) [inline lib/capybara/spec/views/with_animation.erb:66]: If a new view is needed for new tests please create it rather than modifying the existing one.

**Review by twalpole — CHANGES_REQUESTED** [review]
I believe the original intent was for `defer` to handle this use case, but that isn't happening because `defer` is only for external scripts (ignored for inline).  Please remove the `defer` attribute along when making fixes for this PR

File: lib/capybara/spec/views/with_animation.erb (hunk 2 above)
Comment (kuei0221) [reply lib/capybara/spec/views/with_animation.erb]: 🆗

File: spec/shared_selenium_session.rb (hunk 1 above)
Comment (kuei0221) [reply spec/shared_selenium_session.rb]: 🆗

**Comment by twalpole** [comment]
LGTM - thanks

**Pull request merged**